BForBoss.sln.DotSettings

```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
What is this file for?

Reply:
Oh yeah, I explained that in the commit but not the pr description, whoops! It just tells Rider (and I think visual studio? not sure about that though) that Knockback isn't actually a typo, cause the constant warnings were annoying me. If you guys prefer I can undo that and leave it at the user settings level, but I figure there was no harm in making it project wide, since we're using Knockback as a term all over the place.